test/integration/serverless/snapshots/log-proxy

```diff
@@ -0,0 +1,14 @@
+[log] {"message":{"message":"TIMESTAMP UTC | DD_EXTENSION | ERROR | could not forward the request context canceled\n","lambda":{"arn":"arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-STAGE-log-proxy"}},"status":"info","timestamp":TIMESTAMP,"hostname":"arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-STAGE-log-proxy","service":"integration-tests-service","ddsource":"lambda","ddtags":"account_id:601427279990,account_id:601427279990,account_id:601427279990,architecture:x86_64,architecture:x86_64,architecture:x86_64,aws_account:601427279990,aws_account:601427279990,aws_account:601427279990,dd_extension_version:123,dd_extension_version:123,dd_extension_version:123,env:integration-tests-env,env:integration-tests-env,env:integration-tests-env,function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-STAGE-log-proxy,function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-STAGE-log-proxy,function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-STAGE-log-proxy,functionname:integration-tests-extension-STAGE-log-proxy,functionname:integration-tests-extension-STAGE-log-proxy,functionname:integration-tests-extension-STAGE-log-proxy,memorysize:1024,memorysize:1024,memorysize:1024,region:sa-east-1,region:sa-east-1,region:sa-east-1,resource:integration-tests-extension-STAGE-log-proxy,resource:integration-tests-extension-STAGE-log-proxy,resource:integration-tests-extension-STAGE-log-proxy,runtime:nodejs14.x,runtime:nodejs14.x,runtime:nodejs14.x,service:integration-tests-service,service:integration-tests-service,service:integration-tests-service,taga:valuea,taga:valuea,taga:valuea,tagb:valueb,tagb:valueb,tagb:valueb,tagc:valuec,tagc:valuec,tagc:valuec,tagd:valued,tagd:valued,tagd:valued,version:integration-tests-version,version:integration-tests-version,version:integration-tests-version"}
```

Comment:
This looks like it might be an actual issue with the proxy. However, since this feature isn't going to be released yet, I think we can ignore the error for now.